app/src/views/private/components/comment-item-header.vue

```diff
@@ -63,15 +68,15 @@ function useDelete() {
 <template>
 	<div class="comment-header">
 		<v-avatar x-small>
-			<v-image v-if="avatarSource" :src="avatarSource" :alt="userName(activity.user)" />
+			<v-image v-if="avatarSource" :src="avatarSource" :alt="userName(comment.user_created)" />
 			<v-icon v-else name="person_outline" />
 		</v-avatar>
 
 		<div class="name">
-			<user-popover v-if="activity.user && activity.user.id" :user="activity.user.id">
+			<user-popover v-if="comment.user_created && comment.user_created.id" :user="comment.user_created.id">
 				<span>
-					<template v-if="activity.user && activity.user">
-						{{ userName(activity.user) }}
+					<template v-if="comment.user_created && comment.user_created">
+						{{ userName(comment.user_created) }}
```

Comment:
This check seems redundant? The template would not render if `comment.user_created` does not exist. If this is the case we should be able to remove the v-else (private user) as well here as that should never occur either

Reply:
Resolved in 7f1f2ea1d737046c480d563a93c97e885bc29518